_pages/policies.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: Policies
+subtitle: Policies
+permalink: /policies/
+---
+
+All StarCon attendees, speakers, volunteers, and sponsors are asked to abide by these policies.
+
+## Photography Policy
+
+All StarCon attendees, volunteers, speakers, and sponsors will be provided with a badge. Three colors of badges are provided.
+
+- Green means it is okay for photographers to take pictures of you.
+- Blue means photographers should ask permission.
+- Red means photos are not okay (they will not be published).
+
+If no badge is visible, blue will be assumed. Photographers should avoid taking any pictures of anyone wearing a red badge and ask anyone wearing a blue badge. Since it is not possible to always correctly identify badges, if photos are taken of red or blue badge-wearing attendees, we will not publish identifiable photos of those attendees. Anyone in a published photograph in StarCon media can ask to be made unidentifiable in that published photograph.
```

Comment:
"blue": don't just say the colour, specify the policy.
"photographers should avoid" -> photographers will not be

Reply:
said "will avoid" since right after we acknowledge those pictures might be taken but if so they won't be posted. We can't promise no pictures of red badge'd people will be taken